README.md

```diff
@@ -37,6 +126,19 @@ Class.forName("com.salesforce.cdp.queryservice.QueryServiceDriver");
    Connection connection =  DriverManager.getConnection("jdbc:queryService-jdbc:https://login.salesforce.com", properties);
 ```
 
+Create Connection with UserName, ClientId and Private Key (Key Pair authentication) 
+
+```
+Class.forName("com.salesforce.cdp.queryservice.QueryServiceDriver");
+   Properties properties = new Properties();
+   properties.put("user", <UserName>);
+   properties.put("clientId", <Client Id of the connected App>);
+   properties.put("privateKey", <Private Key corresponding to the public digital certificate configured in the connected App>);
```

Comment:
Is this taking file path or the cert string itself?

Reply:
Cert string. After discussion with Palani and team we decided that it is user's responsibility to fetch key from file, trust store, etc. They have to provide the key string here.